src/App.vue

```diff
@@ -371,7 +371,7 @@ export default {
 		flex-shrink: 0;
 
 		.firstrunwizard-header {
-			background-size: var(--image-background-size, 275px, contain);
+			background-size: cover;
```

Comment:
https://github.com/nextcloud/server/blob/8434259b1bfef22adbf30b166043e29763bf9507/apps/theming/lib/Themes/CommonThemeTrait.php#L111

We should drop the variable, it was here for the repeat transparent icon-like background we had in 24 and below.

Reply:
done in https://github.com/nextcloud/server/pull/35547